src/compiler/proposals-and-stabilization.md

```diff
@@ -0,0 +1,314 @@
+a PR is acceptable for the specific contribution (see below), then the approval process can begin.
+
+### Can I work on code experimentally before a approval is gained?
+Of course! You are free to work on PRs or write code. But those PRs should be marked as
+experimental and they should not land, nor should anyone be expected to review them (unless
+folks want to).
+
+## What makes a good proposal?
+A good proposal will address the following:
+
+* **Motivation:** Why is this proposal necessary? What problem does it solve? Why is that problem
+  important?
+* **Design:** What are you proposing?
+* **Implementation notes:** You don't have to talk about the implementation normally, but if there
+  are any key things to note (i.e., it was very invasive to implement), you might note them here.
+* **Precedent, links, and related material:** Have there been similar proposals on other
+  compilers/linkers/tools, like `clang` or `lld`?
+* **Alternatives, concerns, and key decisions:** Were there any alernatives considered? If so, why
+  did you pick this design?
+
+## What proposal/approval do I need?
+This section aims to exhaustively detail which proposal and approval is necessary for any given
+circumstance.
+
+### Internal
+
+- Creating [a notification group](./notification-groups.md)
+  - **Propose using:** PR
+  - **Approve using:** r+
+  - If a team member finds the new group reasonable then they can merge the change adding the group.
+- Significant internal refactorings/changes
+  - **Propose using:** MCP
+  - **Approve using:** Seconding
+  - Describe your proposed refactorings in detail in an MCP - optionally scheduling a steering
+    meeting if more focused discussion is necessary. Once discussion has concluded, a team member
+    may second the proposal
+- Defining/changing small team policies
+  - **Propose using:** MCP
+  - **Approve using:** Seconding
+  - Examples of smaller policy changes where an MCP would be sufficient include our level of
+    support for case-insensitive filesystems or whether the team intend tracking issues to host
+    discussion
+- Defining/changing large team policies
+  - **Propose using:** RFC
+  - **Approve using:** FCP
+  - Larger policy changes requiring an FCP include proposals to the team's structure and
+    membership criteria, etc
+
+### Compiler flags
+
+- Adding a compiler option for internal-use only (e.g. `-Ztreat-bug-as-err`)
+  - **Propose using:** PR
+  - **Approve using:** r+
+  - If a team member finds the new option reasonable then they can merge the change adding the
+    option
+- Adding a simple compiler option with intent to later stabilize
+  - **Propose using:** PR
+  - **Approve using:** r+
+  - Simple options, such as exposing an uncontroversial option from LLVM, can be implemented and
+    merged with just a PR and r+ approval from a reviewer. It will need a full FCP when it is later
+    stabilized
```

Comment:
I suppose it depends on what "simple" means but I think we (at least I) have mostly done MCP in case we want to stabilize a "simple" compiler option[^1][^2].

[^1]: https://github.com/rust-lang/compiler-team/issues/833
[^2]: https://github.com/rust-lang/compiler-team/issues/743

Reply:
Thinking on it more, I think an MCP is reasonable here, changed to that.